tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+envlist = {ceph_ansible2.1}-{jewel}-{xenial_cluster,centos7_cluster}
+skipsdist = True
+
+[testenv]
+whitelist_externals =
+    vagrant
+    bash
+    pip
+    git
+
+passenv=*
+setenv=
+  TOXINIDIR = {toxinidir}
+  ANSIBLE_ACTION_PLUGINS = {toxinidir}/ceph-ansible/plugins/actions
+  # only available for ansible >= 2.2
+  ANSIBLE_STDOUT_CALLBACK = debug
+  centos7_cluster: CEPH_ANSIBLE_SCENARIO_PATH = {toxinidir}/ceph-ansible/tests/functional/centos/7/docker-cluster
+  xenial_cluster: CEPH_ANSIBLE_SCENARIO_PATH = {toxinidir}/ceph-ansible/tests/functional/ubuntu/xenial/docker-cluster
+  ceph_ansible2.1: CEPH_ANSIBLE_BRANCH = stable-2.1
+
+deps=
+  ansible==2.2
```

Comment:
Don't you need to install the test ``requirements.txt`` from ceph-ansible here? If not we won't get testinfra installed or pytest.  Or I suppose you could just define them here directly.